solver/src/solver/http_solver/settlement.rs

```diff
@@ -128,12 +134,14 @@ fn match_prepared_and_settled_orders(
 fn match_prepared_and_settled_amms(
     mut prepared_constant_product_orders: HashMap<usize, ConstantProductOrder>,
     mut prepared_weighted_product_orders: HashMap<usize, WeightedProductOrder>,
+    mut prepared_stable_pool_orders: HashMap<usize, StablePoolOrder>,
```

Comment:
This is starting to feel hairy with the shifts. Not sure what a better solution is yet and doesn't have to be in this PR but feels like a refactor target.

Reply:
I agree, these shifts are 🤮